docs/privacy/data-handling.md

```diff
@@ -0,0 +1,111 @@
+This is particularly important in countries like Germany, where different privacy laws apply
+but also important when it comes to responding to legal subpoenas for the data.
+
+- **When** – will the access to data be provided and for how long?
+It is important to not leave straggling access to data once the engagement is completed, and define a priori the data retention policies.
+
+- **Why** – have you given access to the data?
+This is particularly important to clarify the purpose and any restrictions on usage beyond the intended purpose.
+
+Please use the above guidelines to ensure the data is used only for intended purposes and thereby gain trust.
+It is important to be aware of data handling best practices and ensure the required clarity is provided to adhere to the above 5Ws.
+
+## Handling data in CSE engagements
+
+ Data should never leave customer-controlled environments and contractors and/or other members in the engagement
+ should never have access to complete customer data sets but use limited customer data sets using the following prioritized approaches:
+
+1. Contractors or engagement partners do not work directly with production data, data will be copied before processing per the guidelines below.
+1. Always apply [data minimization](https://www.forbes.com/sites/bernardmarr/2016/03/16/why-data-minimization-is-an-important-concept-in-the-age-of-big-data/#3fb711e91da4)
+principles to minimize the blast radius of errors, only work with the minimal data set required to achieve the goals.
+1. Generate synthetic data to support engagement work. If synthetic data is not possible to achieve project goals,
+request anonymized data in which the likelihood that unique individuals can be re-identified is minimal.
+1. Select a suitably diverse, limited data set, again,
+follow the Principles of Data Minimization and attempt to work with the fewest rows possible to achieve the goals.
+
+Before work begins on data, ensure OS patches are up to date and permissions are properly set with no open internet access.
+
+Developers working on CSE projects will work with our customers to define the data needed for each engagement.
+
+If there is a need to access production data,
+CSE needs to review the need with their lead and work with the customer to put audits in place verifying what data was accessed.
+
+Production data must only be shared with approved members of the engagement team and must not be processed/transferred outside of the customer controlled environment.
+
+Customers should provide CSE with a copy of the requested data in a location managed by the customer.
+The customer should consider turning any logging capabilities on so they can clearly identify who has access and what they do with that access.
+CSE should notify the customer when they are done with the data and suggest the customer destroy copies of the data if they are no longer needed.
+
+Our guiding principles when handling data in an engagement include:
```

Comment:
Suggestion to make this a sub-header to break the content a bit to improve readability.

Reply:
Done